doc/connectivity/bluetooth/overview.rst

```diff
@@ -93,3 +101,24 @@ Bluetooth stack.
     * Local controller support as a virtual HCI driver
 
   * Verified with multiple popular controllers
+
+* LE Audio (Auracast) in Host and Controller
+
+  * Full support for Isochronous channels
+  * Generic Audio Framework
+
+    * Basic Audio Profile
+
+      * Unicast server and client
+      * Broadcast source and sink
+      * Broadcast assistant
+      * Scan delegator
+
+    * Common Audio Profile
+
+      * Acceptor
+      * Initiator and Commander
+
+  * Volume Control Profile, Microphone Control Profile
+  * Call Control Profile, Media Control Profile
+  * Coordinated Set Identification Profile
```

Comment:
These are also part of the Generic Audio Framework (should be indented)

Reply:
Oh right! will fix